Modules/xASL_module_Population.m

```diff
@@ -69,7 +70,15 @@
     warning('Detected no ASL scans, skipping ASL-specific parts of the Population module');
 end
 
-x = xASL_init_InitializeMutex(x, 'Population'); % starts mutex locking process to ensure that everything will run only once
+[x, bLocked] = xASL_init_InitializeMutex(x, 'Population'); % starts mutex locking process to ensure that everything will run only once

```

Comment:
please remove bLocked.

Reply:
Now throughout bLocked -> x.mutex.bAnyModuleLocked

Reply:
Sounds good. Will check and test in detail.